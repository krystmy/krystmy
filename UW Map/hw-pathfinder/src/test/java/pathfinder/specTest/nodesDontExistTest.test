#test finding path of nonexisting nodes

CreateGraph graph

AddNode graph A

FindPath graph A B

FindPath graph B C