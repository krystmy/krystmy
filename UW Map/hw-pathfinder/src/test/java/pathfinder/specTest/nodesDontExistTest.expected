#test finding path of nonexisting nodes

created graph graph

added node A to graph

unknown node B

unknown node B
unknown node C